addon/lib/auth-request.js

```diff
@@ -19,36 +20,26 @@ function pingServer(config, title, data, addon) {
       }
     });
 
-    getMe(headers, config.BASE_URL, function(response) {
-      let id = '';
-      if (response.json) {
-        id = response.json.id;
-      }
-
-      request({
-        url: config.BASE_URL + '/api/metrics/?format=json',
-        content: JSON.stringify(formatEvent(config.TESTPILOT_PREFIX, title, id, data, addon)),
-        headers: headers,
-        onComplete: function(resp) { // eslint-disable-line no-unused-vars
-          // console.error(resp);
-        }
-      }).post();
-    });
+    cb(headers);
   });
 }
 
-function getMe(headers, BASE_URL, cb) {
```

Comment:
Why remove the call to `/api/me`? FWIW, I'm probably going to need it / move it to another spot for feature flipping, since `/api/me` contains the list of all installed experiments and those installations will list feature flags.